go/vt/vtctl/backup.go

```diff
@@ -61,6 +61,8 @@ func init() {
 
 func commandBackup(ctx context.Context, wr *wrangler.Wrangler, subFlags *flag.FlagSet, args []string) error {
 	concurrency := subFlags.Int("concurrency", 4, "Specifies the number of compression/checksum jobs to run simultaneously")
+	allowMaster := subFlags.Bool("allow_master", false, "Allows backups to be taken on master. Warning!! If you are using the builtin backup engine, this will shutdown your master mysql for as long as it takes to create a backup ")
```

Comment:
can we make it so that allow_master is a no-op if it would be harmful?

Reply:
how would we define "harmful"? If there is a way to define that we can look into implementing it.

The idea behind the proposal in the original issue (#4557) was that the user is explicitly overriding the constraint that dis-allows backups from master.

Reply:
@acharis One check I implemented is that even if allow_master is specified with a BackupTablet command, we will first try to do a backup using one of the other types of vttablet(REPLICA/RDONLY/SPARE). Only if none of those is available we will use MASTER. I know this isn't what you asked, but FYI.